playbook/app/pb_kits/playbook/pb_docs/kit_example.html.erb

```diff
@@ -33,13 +33,36 @@
             </li>
           <% end %>
           <li>
-            <a href="#" id="toggle-open" data-toggle="code_example">Code Example</a>
+            <%= pb_rails("button", props: { 
+              text: "Show code", 
+              icon: "copy", 
```

Comment:
@jasperfurniss are we changing out this icon before we ship this? Do you know what the correct icon is?